server/src/Irony.cpp

```diff
@@ -346,8 +356,16 @@ void Irony::complete(const std::string &file,
         }
       }
 
+      // Raise a warning if the candidate may be susceptible to:
+      // https://github.com/Sarcasm/irony-mode/issues/367
+      if (availability == CXAvailability_NotAccessible) {
+        brief = "Irony: Warning: Candidate: `" +
+                prototype +
+                "' maybe be inaccessible due to protected or private access."
+                "\n";
```

Comment:
Agreed overall. I found this useful because it actually changed the size of the mini-buffer which made me look down there. Should I apply a new line only if brief is appended to further down below ?